enterprise/users/src/main/java/io/crate/metadata/sys/SysPrivilegesTableInfo.java

```diff
@@ -87,7 +88,7 @@ public RowGranularity rowGranularity() {
     }
 
     @Override
-    public Routing getRouting(WhereClause whereClause, @Nullable String preference) {
+    public Routing getRouting(WhereClause whereClause, @Nullable String preference, @Nullable SessionContext context) {
```

Comment:
Why is the sessionContext nullable? We've always a session (and context) if a query is executed.

Also, could we reduce the dependency to the User as the other properties of the SessionContext are never used?